apollo-federation/src/connectors/json_selection/methods/public/gte.rs

```diff
@@ -0,0 +1,300 @@
+use apollo_compiler::collections::IndexMap;
+use serde_json_bytes::Value as JSON;
+use shape::Shape;
+use shape::location::SourceId;
+
+use crate::connectors::json_selection::ApplyToError;
+use crate::connectors::json_selection::ApplyToInternal;
+use crate::connectors::json_selection::MethodArgs;
+use crate::connectors::json_selection::VarsWithPathsMap;
+use crate::connectors::json_selection::immutable::InputPath;
+use crate::connectors::json_selection::location::Ranged;
+use crate::connectors::json_selection::location::WithRange;
+use crate::impl_arrow_method;
+
+impl_arrow_method!(GteMethod, gte_method, gte_shape);
+/// Returns true if the applied to value is greater than or equal to the argument value.
+/// Simple examples:
+///
+/// $->echo(3)->gte(3)       results in true
+/// $->echo(4)->gte(3)       results in true
+/// $->echo(2)->gte(3)       results in false
+/// $->echo("a")->gte("b")   results in false
+/// $->echo("c")->gte("b")   results in true
```

Comment:
```suggestion
/// $(3)->gte(3)       results in true
/// $(4)->gte(3)       results in true
/// $(2)->gte(3)       results in false
/// $("a")->gte("b")   results in false
/// $("c")->gte("b")   results in true
```
Can we use less `->echo` in this/future examples?

Here's another syntax that works now (#6784):
```suggestion
/// $(3->gte(3))       results in true
/// $(4->gte(3))       results in true
/// $(2->gte(3))       results in false
/// $("a"->gte("b"))   results in false
/// $("c"->gte("b"))   results in true
```